components/input/input.css

```diff
@@ -0,0 +1,60 @@
+@import "/common.css";
+
+.input-container {
```

Comment:
input-container class you are using while making the example, so it should not be added in your component css.

Reply:
Hi thanks for pointing out, I have fixed it now but can u tell how should we fix it as because we have container for our components and ur point is also right. just need some idea how can we handle this case.